yarn.lock

```diff
@@ -7295,6 +7302,13 @@ axios-retry@^3.0.2:
   dependencies:
     is-retry-allowed "^1.1.0"
 
+axios@0.21.3:
+  version "0.21.3"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.21.3.tgz#f85d9b747f9b66d59ca463605cedf1844872b82e"
+  integrity sha512-JtoZ3Ndke/+Iwt5n+BgSli/3idTvpt5OjKyoCmz4LX5+lPiY5l7C1colYezhlxThjNa/NhngCUWZSZFypIFuaA==
+  dependencies:
+    follow-redirects "^1.14.0"
+
 axios@^0.21.1:
   version "0.21.4"
```

Comment:
yarn doesn't do a good job at this, but if you manually delete both axios from `yarn.lock` and re-run `yarn` it will use a single version, since they are compatible.

Reply:
i couldn't get this to work - yarn puts both versions back every time :( 

i could force this to work by editing the file manually to be:
`axios@^0.21.1, axios@0.21.3:
  version "0.21.3"
  resolved "https://registry.yarnpkg.com/axios/-/axios-0.21.3.tgz#f85d9b747f9b66d59ca463605cedf1844872b82e"
  integrity sha512-JtoZ3Ndke/+Iwt5n+BgSli/3idTvpt5OjKyoCmz4LX5+lPiY5l7C1colYezhlxThjNa/NhngCUWZSZFypIFuaA==
  dependencies:
    follow-redirects "^1.14.0"`
buuuut this didn't feel quite right because we're downgrading axios for the analytics-node package :/ if you think this is better then i can push it :) having more dependencies than we need isn't great but locking to one version also doesn't seem ideal, so i'm not sure which is better